cli/index.ts

```diff
@@ -110,21 +122,30 @@ createYargsCli({
           }
         }
       ],
-      processFn: argv =>
-        init(
+      processFn: async argv => {
+        const result = await init(
           argv["project-dir"],
           {
             warehouse: argv["warehouse"],
             gcloudProjectId: argv["gcloud-project-id"]
           },
           argv["skip-install"]
-        )
+        );
+
+        console.log(commandOutputFmtString, "Directories created:");
+        result.dirsCreated.forEach(dir => console.log(dir));
+        console.log(commandOutputFmtString, "Files written:");
+        result.filesWritten.forEach(file => console.log(file));
+        if (result.installedNpmPackages) {
+          console.log(commandOutputFmtString, "NPM packages successfully installed.");
+        }
+      }
     },
     {
       format: "compile [project-dir]",
       description:
         "Compile the dataform project. Produces JSON output describing the non-executable graph.",
```

Comment:
- Where is the JSON output?
- What do we mean by non-executable graph?

Reply:
* The output gets printed to the console.
* The non-executable graph is the compiled graph, i.e. all of the various references and so on expanded, but no wrapping SQL that would depend on the current state of the warehouse.